tests/jenkins/lib-testers/SignArtifactsLibTester.groovy

```diff
@@ -34,7 +34,7 @@ class SignArtifactsLibTester extends LibFunctionTester {
         assertThat(call.args.platform.first(), notNullValue())
         if(call.args.artifactPath.first().toString().endsWith(".yml")){
             assertThat(call.args.type.first(), notNullValue())
-        } else {
+        } else if(call.args.type.first() != 'maven'){
```

Comment:
I want to call out this change. The following assertion requires that the `sigtype` be required. However, `sigtype` does support a default value of `.asc`.

I added this conditional rather than remove the assertion so that this change has a smaller impact on other tests.